data/test_yaml_stories/stories_and_rules.yml

```
@@ -0,0 +1,51 @@
+rules:
+- rule: rule 1
+  steps:
+  - form: loop_q_form
+  - slot: requested_slot
+    value: some_slot
+  - ...
+  - user: /inform
```

I merged my other PR now, so it should be 
```suggestion
  - intent: inform
```

---

do you mean in the `rules` branch? doesn't seem to be updated yet